camelot/core/files/storage.py

```diff
@@ -98,7 +100,7 @@ def list_files(self, prefix='', suffix='') -> Generator[StoredFile, None, None]:
 
         pattern = f'{prefix}*{suffix}'
         upload_to_path = Path(self.upload_to)
-        return (StoredFile(self, PurePath(path.name)) for path in upload_to_path.glob(pattern))
+        return (StoredFile(self, PurePath(path.name), path.name) for path in upload_to_path.glob(pattern))
```

Comment:
zouden we deze ook nt met self._verbose_name afhandelen, zodat de boel uniform w ?